kubetest/gke.go

```diff
@@ -692,18 +748,22 @@ func (g *gkeDeployer) ensureNat() error {
 	return nil
 }
 
-func (g *gkeDeployer) getRegion(region, zone string) (string, error) {
-	if region != "" {
-		return region, nil
+func (g *gkeDeployer) getRegion() (string, error) {
```

Comment:
Why do we need those changes?

Those doesn't seem to be related to the bastion changes. And if so, I would prefer them to be moved out of this PR to make is as small as possible.

Reply:
Right, after ditching static address handling, getRegion is not used anymore.
Cleaned up.